### ISSUE TITLE
Ability to rename created binary entities

### ISSUE THREAD

**Issue by skynet01** [Odianosen25/Monitor-App#50]
Would be nice to rename binary entities. Right now there is no way to do it, so if you made the mistake during initial set up you are out of luck. 

Proposed solutions: 
- Maybe just updating the text next to Mac address can always be the appropriate name
- If that's too hard maybe it can be specified as a separate line like:

`  known_devices:
    - CC:D2:81:4B:71:F4 Vanessa
       label: "Vanessa is home"
    - CC:D2:81:4D:18:90 Alex 
    - 24:F0:94:7D:EE:83 Guest Tania 
    - 14:D0:0D:7D:F2:3E Guest Anna 
`

**Comment by Odianosen25** [comment]
> Right now there is no way to do it, so if you made the mistake during initial set up you are out of luck.

Actually if you change the name used to setup the MAC address, it should reflect the changes. The issue is, I do use these names to generate the entities and not the MAC. Could add labels as you said,  but will sure be more complicated unless I am missing something. Will see what I can do

**Comment by skynet01** [comment]
I tried changing the name that was used to setup the MAC address but the entities didn't change. Maybe i need to delete some sensors from HA?

**Comment by Odianosen25** [comment]
Yes I thinking deleting the sensors is the right thing, and the app should actually do the clean up for you. I will add that to the list. So if the app sees a new device with MAC, it checks if there are other entities with the same MAC and cleans them out, then loads up the new one.

**Comment by skynet01** [comment]
Yeah exactly and deleting as well, so if I remove a mac address from config it should remove it from the nodes as well. That way adding, removing, and renaming can be all done from Monitor App config file (or UI in the future)
